src/utils/verifyToken.js

```diff
@@ -0,0 +1,11 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+
+dotenv.config();
```

Comment:
can't we use the decode function in the token handler file? you can rename it to verify token

Reply:
the decode function is a middleware function

Reply:
then let's move this function into the token handler and call it in the decode middleware

Reply:
let me add it to the middleware

Reply:
done, added verifyToken to the decode middleware